.clangd

```diff
@@ -0,0 +1,6 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+CompileFlags:
+  Remove: [-march=*]
```

Comment:
```suggestion
  # Workaround for https://github.com/clangd/clangd/issues/1582
  Remove: [-march=*]
```

Reply:
Sure, I have just committed this change.